python/TestHarness/TestHarness.py

```diff
@@ -468,7 +468,7 @@ def buildPBSBatch(self, output, tester):
       return ('QSUB NOT FOUND', '')
     else:
       # Get the PBS Job ID using qstat
-      results = re.findall(r'JOB_NAME: (\w+\d+) JOB_ID: (\d+) TEST_NAME: (\S+)', output, re.DOTALL)
+      results = re.findall(r'JOB_NAME: (\w+) JOB_ID:.* (\d+).*TEST_NAME: (\S+)', output, re.M)
```

Comment:
This change is slightly suspect but so was the original line upon closer inspection. I really don't think you need the "M" option at all since there are no anchors (^,$) in this expression. However, losing the "DOTALL" may or may not be the right thing here. With it if the Pattern repeats, it's possible to have the "dot star" operator match part of an earlier pattern with the ending of a latter pattern. That operator is greedy so that is what normally happens. With it turned off though, your "dot star" operators can never span more than a single line because dot doesn't match newlines by default. 

TL;DR
I suspect the right answer is losing both "M" and "DOTALL".